### ISSUE TITLE
DMFQuest says "Skill too low"

### ISSUE THREAD

**Issue by b-morgan** [ahakola/DMFQuest#3]
I login and DMFQuest says the Faire is available. As I ride into Goldshire the DMFQuest frame pops up but says all my skills are too low.

![image](https://github.com/ahakola/DMFQuest/assets/1547854/326d8f03-e121-4182-9f0e-9cff2d766058)

A "/reload" fixes it.

![image](https://github.com/ahakola/DMFQuest/assets/1547854/df17fa26-6c30-4e0f-94ef-ded1ffa20375)

**Comment by ahakola** [comment]
I'll try to look into this asap

**Comment by ahakola** [comment]
Can't seem to reproduce this. Is this still an issue or just some wierd one time anomaly?

**Comment by b-morgan** [comment]
It happened on every toon I took to the faire, about 5 of them. I've installed 3.01 and have a level 5 toon that has cooking so I'll see what happens with him.

**Comment by b-morgan** [comment]
3.01 shows No professions learned but did not automatically buy the Simple Flour needed for the cooking quest.

![image](https://github.com/ahakola/DMFQuest/assets/1547854/a79c1c57-d13c-4342-b94d-dc69e2fb9331)

I'll leave this toon in Goldshire in case you need anything else.

**Comment by ahakola** [comment]
Is it correct to assume this issue happens every time you login and persist until you reload UI? And the character from the screenshot, it has more professions than just the cooking?

I have other issues to sort for the AutoBuy, so I see what I can do to solve this issue as well while looking into it.

**Comment by b-morgan** [comment]
My monthly procedure is to login on each of my higher level toons and travel to Goldshire,  purchase the items needed for each of their professions, go to the faire, do the profession quests, return to their hearth location, and login to the next toon.

Last month, everything worked as expected. I believe that was with 2.2.11. This month, I've had the issues I have reported. Each toon exhibited the same behavior as shown in the first image in the original post. After a /reload, I got the second image in the original post. Some of the toons auto purchased the required items (as they did with 2.2.11) but some of them did not. I did not keep track of which toon(s) succeeded and which toon(s) failed. 

I will be more careful about documenting their behavior next month.
